bon-macros/src/builder/builder_gen/mod.rs

```diff
@@ -715,12 +717,59 @@ impl BuilderGenCtx {
             }
         };
 
-        let setter_methods = setters
+        let setget_methods = setgets
             .into_iter()
-            .map(|(setter_methods, _)| setter_methods)
+            .map(|(setter_method, _, maybe_getter_method)| {
+                [setter_method, maybe_getter_method.unwrap_or_default()].concat()
+            })
             .concat();
 
-        Ok((setter_methods, items_for_rustdoc))
+        Ok((setget_methods, items_for_rustdoc))
+    }
+
+    fn getter_method(&self, member: &RegularMember) -> Option<TokenStream2> {
+        member.param_getter().map(|()| {
+            // for comments on how this getter formed, please look to setter
+            let member_type = member.norm_ty.as_ref();
+            let member_index = &member.index;
+            let member_state_type =  &member.generic_var_ident;
+            let vis = &self.vis;
+            let member_label = self.members_label(member);
+            let getter_method_name = syn::Ident::new(
+                &format!("get_{}", member.orig_ident),
+                member.orig_ident.span(),
+            );
```

Comment:
Instead of `member.orig_ident`, this should use the `member.setter_method_core_name()`, which take into the account the name override with `#[builder(name = newname)]`. I think it makes sense to reuse that override for the getter as well. Note that we'll need to rename the `setter_method_core_name` to `accessor_method_core_name` to make it generic and shared between setters and getters.